packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -55,7 +55,7 @@ See [Integrate with Endpoint Detection and Response solutions
 
 ### Security Context
 
-You can specify certain [rule conditions](/docs/reference/api/policy/#conditions) in [app sign-on policies](/docs/reference/api/policy/#app-sign-on-policy) using expressions based on the Security Context of the app sign-on request. Security Context is made up of the [risk level](https://help.okta.com/okta_help.htm?id=csh-risk-scoring) and the matching [User behaviors](https://help.okta.com/okta_help.htm?id=ext_proc_security_behavior_detection) for the request.
+You can specify certain [rule conditions](/docs/reference/api/policy/#conditions) in [authentication policies](/docs/reference/api/policy/#authentication-policy) using expressions based on the Security Context of the app sign-on request. Security Context is made up of the [risk level](https://help.okta.com/okta_help.htm?id=csh-risk-scoring) and the matching [User behaviors](https://help.okta.com/okta_help.htm?id=ext_proc_security_behavior_detection) for the request.
```

Comment:
Is the heading "App sign-on policy" going to change to "Authentication policy" in the `/docs/reference/api/policy` page?

Reply:
Yes, that's part of the other branch I have going with the Dev team.